dotnetv3/EventBridge/Actions/HelloEventBridge.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[EventBridge.dotnetv3.HelloEventBridge]
+using Microsoft.Extensions.Hosting;
+using Amazon.EventBridge;
+using Amazon.EventBridge.Model;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace EventBridgeActions;
+
+public static class HelloEventBridge
+{
+    static async Task Main(string[] args)
+    {
+        // Use the AWS .NET Core Setup package to set up dependency injection for the Amazon EventBridge service.
+        // Use your AWS profile name, or leave it blank to use the default profile.
```

Comment:
I'm not sure where I'd fill in a profile name...or is this obvious to a .NET customer?

Reply:
I updated the code to make it more clear.